domain/tutoringModel/model.py

```diff
@@ -29,15 +32,49 @@ def serialize(self):
             'calculated_on': self.calculated_on
         }
 
-    def get_learning_path(self, student_id, learning_style, algorithm):
-        if algorithm == "Graf":
+    def get_learning_path(self,
+                          student_id,
+                          learning_style,
+                          algorithm,
+                          list_of_les):
+        if algorithm == "graf":
             path = graf.GrafAlgorithm(
                 student_id=student_id, learning_style=learning_style)
             temp = path.get_learning_path(input_learning_style=learning_style)
             self.path = ", ".join(temp)
+        elif algorithm == "aco":
+            list_of_les_classifications = self.prepare_le_for_aco(list_of_les)
+            coordinates = get_coordinates(
+                learning_style, list_of_les_classifications)
+            path = aco.AntColonySolver()
+            result = path.solve(list(coordinates.items()))
+            le_path = ""
+            for ele in result:
+                le_path = le_path + ele[0] + ', '
+            self.path = le_path[:-2]
         else:
             raise err.NoValidAlgorithmError()
 
+    def prepare_le_for_aco(self, list_of_les):
+        lz_in_list = False
+        list_of_les_classifications = []
+        for le in list_of_les:
+            if le['classification'] == 'KÜ':
```

Comment:
Vllt sollten wir ein Mapping/Enum, analog zum glossar machen, damit wir hardcoded strings durch compiler checks erweit
```suggestion
            if le['classification'] == someenum.kuerzuebersicht:
```ern können.